### DISCUSSION TITLE
Become a member of FearlessTech

### DISCUSSION THREAD

**Discussion by 4-Leafs-Code** [FearlessTech/FearlessTechDiscussions#4] [Show and tell]
Would you like to become a member of FearlessTech? Please explain how you have contributed to one of our projects or FearlessTech. Any kind of valid contribution will be accepted and you'll be able to show our awesome organization from your GitHub profile. Rather you a programing, mentoring, or just telling your friends about us. This is not a requirement but I'd personally like to request you try to make a contribution about once a month, even if it is to answer a quick question. We strive to be a friendly and responsive group of collaborators who are all learning from and teaching one another.

Your membership role will be assigned to match your skillset starting at the lowest level needed and will be increased based on the quality of your contributions. If you would not mind sharing in what areas you excel, this would give us an idea of how best we can help each other.

**Comment by naji0329** [comment]
I am a web developer.
I have developed one project which is going to lunch in this year.
You can check with this link.
https://craftbook.io
Thanks.

**Reply by 4-Leafs-Code** [reply]
@NAJI0329 I love this: Craftbook brings together creators, curators, influencers, fans and crypto enthusiasts on a single platform to collaborate and engage on a wider ecosystem. This would be a great project for this community to collaborate on if it was open source. Will you eventually charge for people to use this?

It is very much the same idea Im aiming for with FearlessTech, except this is open source. I tried to sign up to but I have yet to receive the verification email but this could be this facility's firewall blocking it.  Let me know when/if you would like me test it out and provide feedback. 

Also, I want to eventually make a members highlights  on the FearlessTech website (gh page). If you'll approve, I'd like to show case your work. You could build the showcase yourself or I can do it, which ever you prefer.

**Reply by 4-Leafs-Code** [reply]
To add to @NAJI0329 comment: Craftbook is an initiative for Customized Generative Art that aims to gamify the world of digital art and provide a boundless platform for creativity. This is a create-to-earn platform that provides the power to every individual to create anything that engages their community. Easy to use and inclusive platform where user and creator collaborate to try out unique ideas.

It looks like it will be a very fun tool, check it out!

**Comment by juliusjoseph5269-eng** [comment]
Hello

My name is Julius Joseph, and I’m a Computer Science student interested in cybersecurity, web development, and software development.
My main interests are web application security, penetration testing, and building practical websites and software applications. I’m currently looking to grow my practical experience and explore opportunities to work on real-world projects, including internships, freelance work, and junior roles.
I joined this community to learn from others, share ideas, contribute to open-source projects, collaborate with other developers, and gain more hands-on experience.
I’m happy to be part of the community and look forward to connecting with everyone and contributing wherever I can. 🚀
